modules/administration-guide/pages/configuring-che-storage-strategy.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: CONCEPT
+:description: Configuring the storage strategy in {prod-short} 
+:keywords: administration guide, configuring, {prod-short}, storage, strategy
+:navtitle: Configuring the storage strategy in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-strategy.adoc
+
+[id="configuring-{prod-id-short}-storage-strategy"]
+= Configuring the storage strategy in {prod-short}
+
+{prod-short} can be configured to provide persistent or non-persistent storage to workspaces by selecting a storage strategy. The selected storage strategy will be applied to all newly created workspaces by default. Users can opt for a non-default storage strategy for their workspace in their xref:end-user-guide:requesting-persistent-storage-for-workspaces.adoc[devfile] or through the xref:end-user-guide:url-parameter-for-the-workspace-storage.adoc[URL parameter].
```

Comment:
While working on this, I realized that the documentation on requesting persistent storage for workspaces with a devfile (`requesting-persistent-storage-for-workspaces.adoc`) also should probably be updated to reference the [DevWorkspace Operator storage strategy attribute](https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#configuring-persistent-storage-used-for-a-devworkspace)

Reply:
I've created https://github.com/eclipse/che/issues/22316 to track this issue and determine if it is valid, as it may not be something Che users need to know about.